internal/types/subtypes/interceptor_ext_test.go

```diff
@@ -0,0 +1,291 @@
+package subtypes_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/boundary/internal/gen/testing/attribute"
+	"github.com/hashicorp/boundary/internal/types/subtypes"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/testing/protocmp"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+func TestAttributeTransformerInterceptor(t *testing.T) {
+	cases := []struct {
+		name             string
+		req              interface{}
+		handlerRes       interface{}
```

Comment:
Can we name this `handlerResp` to disambiguate from `handlerResult`? I initially read "result" not "response".